tests/vlan/test_vlan.py

```diff
@@ -43,26 +43,12 @@ def vlan_ports_list(cfg_facts, ptfhost):
 
     pvid_cycle = itertools.cycle(vlan_id_list)
 
-    # when running on t0 we can use the portchannel members
-    if config_portchannels:
```

Comment:
The portchannel members were added intentionally.  Please refer to the original PR
https://github.com/Azure/sonic-mgmt/pull/375
https://github.com/Azure/sonic-mgmt/blob/master/docs/VLAN-trunk-test-plan.md

Reply:
Thanks Neetha. The doc is helpful. I will make an update to this PR.

Reply:
Please help to review. Thanks!

> The portchannel members were added intentionally. Please refer to the original PR
> #375
> https://github.com/Azure/sonic-mgmt/blob/master/docs/VLAN-trunk-test-plan.md